### PR TITLE
Adding more columns to report

### PR DESCRIPTION
**Pull request by SilviaAmAm**
Part of #55 

**What's new**
- Additional information is added to the report of destruction

**What's still missing**
The columns Beschrijving and Opmerkingen are not optional yet, this is implemented in PR https://github.com/maykinmedia/archiefvernietigingscomponent/pull/147.

### CODE REVIEW

File: src/archiefvernietigingscomponent/destruction/tasks.py [hunk 1]
```diff
@@ -99,13 +99,21 @@ def process_list_item(list_item_id):
             template="destruction/logs/item_destruction_succeeded.txt",
             extra_data={"zaak": zaak["identificatie"]},
         )
+        try:
+            resultaat = fetch_resultaat(zaak["resultaat"])
```
Comment (annashamray) [inline src/archiefvernietigingscomponent/destruction/tasks.py:103]: I think `resultaat` should be already destroyed when you fetch it

File: src/archiefvernietigingscomponent/destruction/service.py [hunk 2]
```diff
@@ -146,6 +146,19 @@ def get_resultaat(zaak_url: str) -> Optional[dict]:
     return resultaat
 
 
+def fetch_resultaat(resultaat_url: str) -> dict:
```
Comment (annashamray) [inline src/archiefvernietigingscomponent/destruction/service.py:149]: this function and `get_resultaat` share a lot of code. Could you use this function inside `get_resultaat` to make it DRYer?

File: src/archiefvernietigingscomponent/destruction/tasks.py [hunk 3]
```diff
@@ -99,13 +99,21 @@ def process_list_item(list_item_id):
...
         list_item.extra_zaak_data = {
             "identificatie": zaak["identificatie"],
             "omschrijving": zaak.get("omschrijving") or "",
             "toelichting": zaak.get("toelichting") or "",
             "startdatum": zaak["startdatum"],
             "einddatum": zaak.get("einddatum") or "",
             "zaaktype": zaak["zaaktype"],
+            "verantwoordelijke_organisatie": zaak["verantwoordelijkeOrganisatie"],
+            "resultaat": resultaat,
+            "relevante_andere_zaken": zaak.get("relevanteAndereZaken") or [],
```
Comment (annashamray) [inline src/archiefvernietigingscomponent/destruction/tasks.py:116]:
```suggestion
            "relevante_andere_zaken": zaak.get("relevanteAndereZaken", []),
```

**Review by annashamray — CHANGES_REQUESTED** [review]

File: src/archiefvernietigingscomponent/destruction/tasks.py (hunk 1 above)
Comment (joeribekker) [reply src/archiefvernietigingscomponent/destruction/tasks.py:103]: Good catch!

**Review by joeribekker — APPROVED** [review]

**Pull request merged**